mne/evoked.py

```diff
@@ -115,6 +117,37 @@ def __init__(self, fname, condition=None, baseline=None, proj=True,
             self.apply_proj()
         self.data = rescale(self.data, self.times, baseline, copy=False)
 
+    @verbose
+    def apply_baseline(self, baseline, verbose=None):
+        """Baseline correct evoked data
+
+        Parameters
+        ----------
+        baseline : tuple of length 2
```

Comment:
This could probably default to `(None, 0)`, same with `Epochs.apply_baseline`. The latter is backward-compatible change so we can do it. That would make them consistent with `Epochs.__init__`. But we should wait to see if others agree.